bundles/org.openhab.ui.homebuilder/web/src/restItems.js

```diff
@@ -0,0 +1,210 @@
+import { floors, devices, DEVICES_SUFFIX } from './definitions'
+
+export const GROUP_PREFIX = 'g';
+
+export function addFloors(floor, model) {
+    let items = [];
+    if (model.floorsCount > 1) {
+        items.push({
+            type: 'Group',
+            name: floor.abbr,
+            label: floor.name || floor.value,
+            category: model.itemsIcons ? floor.icon : '',
+            groupNames: ['Home'],
+            entryType: 'floor'
+        });
+    }
+
+    return items;
+}
+
+export function addRooms(floor, model) {
+    let items = [];
+
+    if (floor && floor.value && !_.isUndefined(model[floor.value])) {
```

Comment:
if _ lodash is available globally, please review other code and change `typeof` accordingly.

Reply:
It is everywhere now except `Collapse` and custom `Multiselect` components